mergebot_backend/merge.py

```diff
@@ -260,6 +279,17 @@ def flush_queue(self):
         """
         pr = self.work_queue.get_nowait()
         self.publisher.publish_dequeue(item_id=pr.get_num())
+        pr.post_commit_status(
+            state=github_helper.COMMIT_STATE_ERROR,
+            url=MERGEBOT_ITEM_URL.format(
+                name=self.config.name, number=pr.get_num()),
+            description='MergeBot shutdown; resubmit when mergebot is back up.',
```

Comment:
I'll update the URL to the top-level MergeBot page which indicates whether MergeBot is back up, but all of the URLs on these point to the MergeBot frontend.